### ISSUE TITLE
Update little footer with the "#1 in the US for innovation" badge

### ISSUE THREAD

**Issue by rraub** [gios-asu/ASU-Web-Standards-Wordpress-Theme#170]
One example I've seen looks like this:
![screen shot 2015-10-09 at 9 47 00 am](https://cloud.githubusercontent.com/assets/295804/10399984/fbc0e7fa-6e6a-11e5-81e4-fe468d76f1cb.png)

I think the look can be improved, right now its kinda clunky..
Examples: https://uto.asu.edu/
Also note the text "Jobs At ASU" has been renamed to "Jobs" in the footer.

**Comment by rraub** [comment]
I think the badge should be clickable, and maybe reword the text 
```#1 in the U.S. for innovation``` to 
```ASU is #1 in the U.S. for innovation```

**Comment by idmontie** [comment]
Is the footer something we can reference like the header? I know that slows it down, but it is a process to get new updates pushed for things like this.

**Comment by rraub** [comment]
According to their site no, https://brandguide.asu.edu/web-standards/enterprise/global-asu-footer 
They don't event have these new changes documented.

**Comment by idmontie** [comment]
Will close in #176, but I think we should keep thinking about how we can style it.

**Comment by rraub** [comment]
I was hoping to ignore this until we could pitch our alternative layout to the group, or until they could formalize the standard for this addition.

**Comment by idmontie** [comment]
It was verified to be "final" in the web standards meeting today.

On Wed, Oct 21, 2015 at 2:33 PM, Ryan Raub <notifications@github.com> wrote:

> I was hoping to ignore this until we could pitch our alternative layout to
> the group, or until they could formalize the standard for this addition.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gios-asu/ASU-Web-Standards-Wordpress-Theme/issues/170#issuecomment-150030599>
> .
>



-- 
Best regards,

*Ivan Montiel*
*Web Application Developer*

P.O. Box 875402 | Tempe, Arizona | 85287-5402
Main: 480-965-2975
sustainability.asu.edu

**Comment by rraub** [comment]
There was a web standards meeting today? has this been a recurring meeting I have just never known about because its on Wednesdays? 

Did you pitch our alternative layouts in the meeting?

**Comment by idmontie** [comment]
It was a non-recurring web standards meeting. It was mainly for
non-academic units at ASU. I asked whether the footer was finalized and
brought up the issues of the layout as it is currently. Jill said that for
all intents and purposes, it was final.

On Wed, Oct 21, 2015 at 2:38 PM, Ryan Raub <notifications@github.com> wrote:

> There was a web standards meeting today? has this been a recurring meeting
> I have just never known about because its on Wednesdays?
>
> Did you pitch our alternative layouts in the meeting?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gios-asu/ASU-Web-Standards-Wordpress-Theme/issues/170#issuecomment-150032467>
> .
>



-- 
Best regards,

*Ivan Montiel*
*Web Application Developer*

P.O. Box 875402 | Tempe, Arizona | 85287-5402
Main: 480-965-2975
sustainability.asu.edu
